english/challenge-marshmellow-to-rock-off/castle-route/narnia.md

```diff
@@ -0,0 +1,26 @@
+You forget about the existence of that white light. Someone did say "don't go towards the light." 
+In your frantic scramble, you rush to the door and reach for the handle. But it won't budge. 
+You push and pull and it doesn't budge. The color drains from your face as you smell the sweet, 
+sweet savory goodness of burning marshmallows edging closer and closer towards you. 
+You turn around and your stomach does a backflip as you witness the slow but steady crawl of melted white goo towards you. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.